tools/docker-compose/entrypoint.sh

```diff
@@ -32,4 +32,4 @@ else
     export SDB_NOTIFY_HOST=$(ip route | head -n1 | awk '{print $3}')
 fi
 
-exec $@
+exec "$@"
```

Comment:
This seems like it would break when starting the container with a command that has multiple arguments, which I do pretty frequently. Please revert this.